ee/tabby-webserver/src/service/repository.rs

```diff
@@ -37,6 +41,62 @@ impl RepositoryService for DbConn {
             .await?;
         Ok(true)
     }
+
+    async fn search_files(
```

Comment:
If implementation makes sense to you, please add a unit test and send PR to this branch @boxbeam